ts/features/wallet/onboarding/common/searchBank/SearchBankScreen.tsx

```diff
@@ -22,8 +23,10 @@ import SectionStatusComponent from "../../../../../components/SectionStatusCompo
 import { IOStyles } from "../../../../../components/core/variables/IOStyles";
 import { SearchBankComponent } from "./SearchBankComponent";
 
+type MethodType = "bancomatPay" | "bancomat";
```

Comment:
What about this?
`type MethodType = Extract<SectionStatusKey, "bancomatPay" | "bancomat">;`
in this way we can avoid `getSectionName`

Reply:
We have `bancomatpay` as `SectionStatusKey` but `bancomatPay` (camelCase) as MethodType. 
Changing the keys in the `SectionStatusKey` would break the previous app version (we use camelCase but this config have snake case)
What you think?

Reply:
Agree.
I didn't notice the camelCase on `bancomatPay`